### ISSUE TITLE
dataset problem

### ISSUE THREAD

**Issue by YuyingF** [matciotola/Lambda-PNN#2]
Hello,

I have been following your method to process some images in TIFF format. However, when I attempted to test it, I encountered an issue. The error occurred in the input_propocessing.py file, specifically at the line where it attempts to concatenate two tensors: img_in = torch.cat((ms_exp, pan), dim=1) / max_value.

The sizes of the ms_exp and pan tensors are as follows:

ms_exp: torch.Size([1, 3, 1024, 1024])
pan: torch.Size([1, 1, 3, 1024, 1024])
As I am new to Deep Learning, I’m having trouble understanding why this error is occurring and how to resolve it. Could you please provide some guidance or suggestions to help me troubleshoot this issue? I would greatly appreciate your assistance.

Thank you very much.

**Comment by matciotola** [comment]
Hi @YuyingF 
I saw the error! I do not know why you have a Tensor shape of  5-D, with 3 channels for the PAN. The PAN should be of shapes [1, 1, 1024, 1024]. Please check your images, you have wrong shapes!
